ci-operator/step-registry/cucushift/installer/rehearse/baremetalds/assisted/ovn/hypershift-agent/deprovision/cucushift-installer-rehearse-baremetalds-assisted-ovn-hypershift-agent-deprovision-chain.yaml

```diff
@@ -0,0 +1,7 @@
+chain:
+  as: cucushift-installer-rehearse-baremetalds-assisted-ovn-hypershift-agent-deprovision
+  steps:
+  - chain: hypershift-agent-destroy
+  - chain: cucushift-installer-rehearse-baremetalds-assisted-deprovision
+  documentation: |-
+    Destroy IPI cluster on Baremetal (via devscripts)
```

Comment:
Agent is not IPI, you could document it as baremetal install with assisted service